iceberg-handler/pom.xml

```diff
@@ -16,41 +16,46 @@
 
     <properties>
         <hive.path.to.root>..</hive.path.to.root>
-        <iceberg-api.version>0.11.0</iceberg-api.version>
         <kryo-shaded.version>4.0.2</kryo-shaded.version>
-        <avro.iceberg.version>1.9.2</avro.iceberg.version>
+        <avro.iceberg.test.version>1.9.2</avro.iceberg.test.version>
```

Comment:
Maybe it is just a little bit misleading name. We will use the 1.9.2 avro for Iceberg as well too.
Shall we just rename all the iceberg specific versions to:
- iceberg.kryo.version
- iceberg.avro.version
?

Reply:
Good idea, done